packages/shorebird_code_push_protocol/lib/src/messages/create_release_artifact/create_release_artifact_request.dart

```diff
@@ -36,6 +37,10 @@ class CreateReleaseArtifactRequest {
   /// The hash of the artifact.
   final String hash;
 
+  /// The name of the file.
+  /// TODO(NEXT_API): make this non-nullable.
```

Comment:
```suggestion
  /// TODO(bryanoltman): make this non-nullable in the next major release.
```

Reply:
Since we've got an impending version bump, I'm just going to go ahead and make this non-optional with the assumption that we won't release a version between now and the update.